frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -812,6 +811,7 @@ const HostDetailsPage = ({
           onRefetchHost={onRefetchHost}
           renderActionDropdown={renderActionDropdown}
           osSettings={host?.mdm.os_settings}
+          osVersionRequirement={getOSUpdatesFromMDMConfig(host.platform)}
```

Comment:
nit, might be nice to replace the function name to match the prop name

Reply:
sgtm